magenta/music/musicxml_parser.py

```diff
@@ -374,8 +379,13 @@ def __init__(self, xml_measure, state):
     self.tempos = []
     self.time_signature = None
```

Comment:
Yeah, I think it's fine not to support these edge cases. To make it clear what we do support, I'd suggest adding checks for these circumstances and raising an exception if one occurs.